recipes-kernel/linux/files/5.x-kernel/0004-USB-serial-cp210x-add-support-for-software-flow-cont.patch

```diff
@@ -0,0 +1,204 @@
+From a77439d90025b672b54b9c282f1d91bbb5b229d3 Mon Sep 17 00:00:00 2001
+From: Wang Sheng Long <shenglong.wang.ext@siemens.com>
+Date: Mon, 18 Jan 2021 12:13:26 +0100
+Subject: [PATCH 04/12] USB: serial: cp210x: add support for software flow
+ control
```

Comment:
I think you didn't update the recipe yet to pull this (and drop the downstream patch).

Reply:
for kernel 5.4.74,using the upstream patch would has lots of conflicts,if we'd like to use upstream related patch,we should pick server commits from upstream.  So in currently version,using this patch I think is ok

Reply:
Should look better with 5.10 but, yes, it will likely need more backports. Using upstream patches generally the better option because they have been reviewed and tested more broadly.